roles/openshift_logging_elasticsearch/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 ### Common settings
-openshift_logging_elasticsearch_image_pull_secret: "{{ openshift_hosted_logging_image_pull_secret | default('') }}"
+openshift_logging_elasticsearch_image_pull_secret: "{{ openshift_hosted_logging_image_pull_secret | default(openshift_logging_image_pull_secret) }}"
```

Comment:
can you instead set this as? 
`openshift_logging_elasticsearch_image_pull_secret: "{{ openshift_logging_image_pull_secret | default('') }}"`

Reply:
I am happy with omitting the `*_hosted_*` variable. But with `openshift_logging_image_pull_secret` having a default set in the role `openshift_logging`, is the `default()` necessary?

Reply:
to me its a way to ensure that this has a value in the case where a user nils out `openshift_logging_image_pull_secret` -- we would still end up with a value for `openshift_logging_elasticsearch_image_pull_secret`